### PR TITLE
Source Hubspot: add new fields to schema

### LINKED ISSUE #28467: Source Hubspot: add new fields to schemas
Add missing fields to schemas for streams in Source Hubspot

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
Add changes in scope of this epic:
https://github.com/airbytehq/airbyte/issues/28202

## How
* Add fields to schema of `campaigns` stream (`lastProcessingFinishedAt`, `lastProcessingStateChangeAt`, `lastProcessingStartedAt`, `processingState`)
* Add fields to schema of `forms` stream (`formType`)
* Add fields to schema of `contact_lists` stream (`ilsFilterBranch`, `internal`, `authorId`, `limitExempt`, `teamIds`)
* Add fields to schema of `email_events` stream (`subscriptions`, `bcc`, `cc`)
* Add fields to schema of `engagements` stream (`queueMembershipIds`, `scheduledTasks`)
* Add fields to schema of `email_subscriptions` stream (`order`, `businessUnitId`, `internal`, `internalName`, `category`, `channel`)
* * Add fields to schema of `subscription_changes` stream (`normalizedEmailId``)
* Add fields to schema of `marketing_emails` stream (`publishedByEmail `, `sections`, `author`, `isCreatedFomSandboxSync`, `rssEmailUrl`, `teamPerms`, `securityState`, `isInstanceLayoutPage`, etc.)
* Add fields to schema of `property_history` stream (`source-vids`)
* Add fields to schema of `workflows` stream (`lastUpdatedByUserId`, `contactCounts`, `description`, `originalAuthorUserId`, `migrationStatus`, `updateSource`, `creationSource`)
* Add fields to schema of `goals` stream (`hs_kpi_value_last_calculated_at`)
* Update expected records of streams `deals`, `companies`, `pets`, `cars`

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`efb3508fa1`](https://github.com/airbytehq/airbyte/pull/28349/commits/efb3508fa1ff7f18db0a530af7f20f95da8419f2)) - ❌

⏲️  Total pipeline duration: 10mn32s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ❌        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689580213/efb3508fa1ff7f18db0a530af7f20f95da8419f2/source-hubspot/1.0.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`00d48a2dbe`](https://github.com/airbytehq/airbyte/pull/28349/commits/00d48a2dbe30451e09d1e28b644cd6377bf382da)) - ❌

⏲️  Total pipeline duration: 51.80s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ❌        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689680286/00d48a2dbe30451e09d1e28b644cd6377bf382da/source-hubspot/1.0.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`c3c7b2bb9e`](https://github.com/airbytehq/airbyte/pull/28349/commits/c3c7b2bb9e96b74abaf1ad974e158bdf8f7b7619)) - ❌

⏲️  Total pipeline duration: 53.80s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ❌        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689684653/c3c7b2bb9e96b74abaf1ad974e158bdf8f7b7619/source-hubspot/1.0.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`1381e355df`](https://github.com/airbytehq/airbyte/pull/28349/commits/1381e355df9125eaae04f6038f86a35eb876f275)) - ❌

⏲️  Total pipeline duration: 13mn22s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689684817/1381e355df9125eaae04f6038f86a35eb876f275/source-hubspot/1.0.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`c7af564854`](https://github.com/airbytehq/airbyte/pull/28349/commits/c7af56485485de025d327118ee7e9587e8f37806)) - ✅

⏲️  Total pipeline duration: 20mn14s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689699312/c7af56485485de025d327118ee7e9587e8f37806/source-hubspot/1.0.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`94359c15b9`](https://github.com/airbytehq/airbyte/pull/28349/commits/94359c15b93c231eaab517e2a0e66203ee57d625)) - ❌

⏲️  Total pipeline duration: 13mn08s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689753675/94359c15b93c231eaab517e2a0e66203ee57d625/source-hubspot/1.0.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`128a60ad3a`](https://github.com/airbytehq/airbyte/pull/28349/commits/128a60ad3a8bd1b0136f0baa0f941326bba1d725)) - ❌

⏲️  Total pipeline duration: 11mn18s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689754552/128a60ad3a8bd1b0136f0baa0f941326bba1d725/source-hubspot/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`6812ffa120`](https://github.com/airbytehq/airbyte/pull/28349/commits/6812ffa1200887293391334e91bc11d99550691a)) - ✅

⏲️  Total pipeline duration: 10mn15s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689756031/6812ffa1200887293391334e91bc11d99550691a/source-hubspot/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json [hunk 1]
```diff
@@ -42,6 +42,95 @@
...
+        "updatedAt": {"type" : ["null", "integer"]},
+        "updatedByUser": {
+          "properties" : {
+            "userEmail": {"type" : ["null", "string"]},
+            "userId": {"type" : ["null", "integer"]}
+          }
+        }
+      }
+    },
+    "creationSource": {
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json:110]: `type` field is missing here as we as in many other properties. I don't think this is a valid definition

**Review by davydov-d — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json (hunk 1 above)
Comment (arsenlosenko) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json:110]: good point, I'll go through schemas once again to see if some type declarations were missed

Comment (arsenlosenko) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json:113]: updated type declarations here and in other schemas

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`c331bf7722`](https://github.com/airbytehq/airbyte/pull/28349/commits/c331bf772284295469eb65aa1fd9a69e1b560bc9)) - ✅

⏲️  Total pipeline duration: 12mn47s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689764039/c331bf772284295469eb65aa1fd9a69e1b560bc9/source-hubspot/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Review by lazebnyi — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/email_events.json [hunk 2]
```diff
@@ -8,6 +8,12 @@
     "appName": {
       "type": ["null", "string"]
     },
+    "bcc": {
+      "type": ["null", "array"]
+    },
+    "cc": {
+      "type": ["null", "array"]
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/email_events.json:15]: can we specify array items types?

**Review by davydov-d — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-hubspot test report (commit [`c33c7376fc`](https://github.com/airbytehq/airbyte/pull/28349/commits/c33c7376fc37f5664764ee6603bac8fecfe81fe4)) - ✅

⏲️  Total pipeline duration: 10mn40s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-hubspot/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-hubspot docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-hubspot_to-silver/1689840081/c33c7376fc37f5664764ee6603bac8fecfe81fe4/source-hubspot/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/email_events.json (hunk 2 above)
Comment (arsenlosenko) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/email_events.json:15]: checked if there are any values in this fields in records we receive from test account, these fields are empty right now, so I can't specify the exact type of their items

**Pull request merged**